ConservationInternational/resilienceatlas
# feat: API endpoint for getting static pages

API endpoint which allows to obtain information about Static page -- in our case about `About` page

## Testing instructions

Make sure that you have "some" data for Static Page. Open swagger documentation and provide `slug` of your static page as part of path. Double check that data returned via API corresponds to data visible at backoffice.

## Tracking

https://vizzuality.atlassian.net/browse/RA2-190
